library.properties

```diff
@@ -1,7 +1,8 @@
 name=UbidotsMQTT
-version=1.1.0
+version=1.1.2
 license=MIT
 author=Ubidots <devel@ubidots.com>, Jose Garcia <jose.reyes@ubidots.com>
```

Comment:
Change this email to jose.garcia@ubidots.com

Reply:
@woakas  solved